README.md

```diff
@@ -45,8 +45,8 @@ widely used across the Earth, Ocean, and Planetary sciences and beyond.
 - Build a Pythonic API for GMT.
 - Interface with the GMT C API directly using ctypes (no system calls).
 - Support for rich display in the Jupyter notebook.
-- Integration with the [PyData ecosystem](https://pydata.org/): `numpy.ndarray` or `pandas.DataFrame` for
-  data tables, `xarray.DataArray` for grids, and `geopandas.GeoDataFrame` for geographical data.
+- Integration with the [Scientific Python ecosystem](https://scientific-python.org/): `numpy.ndarray` or
```

Comment:
I feel it should be "scientific Python ecosystem" following https://scientific-python.org/about/ and https://www.nature.com/articles/s41586-020-2649-2.

Reply:
Yes, I was also wondering which version we want to use, as I found all three "Scientific Python Ecosystem", "Scientific Python ecosystem", and "scientific Python ecosystem". But going with the one used on the official website makes sense.